frontend/src/container/QueryBuilder/filters/ReduceToFilter/ReduceToFilter.tsx

```diff
@@ -0,0 +1,26 @@
+import { Select } from 'antd';
+import React, { memo } from 'react';
+// ** Types
+import { EReduceOperator } from 'types/common/queryBuilder';
+import { SelectOption } from 'types/common/select';
+
+import { ReduceToFilterProps } from './ReduceToFilter.interfaces';
+
+export const ReduceToFilter = memo(function ReduceToFilter({
+	query,
+	onChange,
+}: ReduceToFilterProps): JSX.Element {
+	const options: SelectOption<string, string>[] = Object.values(
```

Comment:
always avoid inline logic

Reply:
it is just preparing options for select, nothing specific. Have we create each time new function that prepares for us options or other data?